src/jvmTest/kotlin/bdd/InventoryLocation.kt

```diff
@@ -0,0 +1,6 @@
+package bdd
+
+data class InventoryLocation(
+    val slot: Int,
+    val page: Int
```

Comment:
Inventory is a misleading name. It should probably be "ToolbarLocation".

Inventory is where the building materials are... this is just UI thing, what is available on the numbers.

Reply:
Got it, my bad.